lms/templates/course.html

```diff
@@ -0,0 +1,42 @@
+<%def name="online_help_token()"><% return "course" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+from six import text_type
+%>
+<%page args="course" expression_filter="h"/>
+<article id="${course.id}" role="region" aria-label="${course.display_name_with_default}">
+  <a href="${reverse('about_course', args=[text_type(course.id)])}">
+    <div class="course-info" aria-hidden="true">
+        <span class="course-code">${course.display_number_with_default}</span>
+        <span class="course-organization">${course.display_org_with_default}</span>
+        <span class="course-title">${course.display_name_with_default}</span>
+      <%
+      if course.start is not None:
+          course_date_string = course.start.strftime('%Y-%m-%dT%H:%M:%S%z')
+      else:
+          course_date_string = ''
+      %>
+      % if course.advertised_start is not None:
+          <div class="course-date" aria-hidden="true">${_("Starts")}: ${course.advertised_start}</div>
+      % else:
+          <div class="course-date localized_datetime" aria-hidden="true" data-format="shortDate" data-datetime="${course_date_string}" data-string="${_("Starts: {date}")}"></div>
+      % endif
+    </div>
+    <div class="sr">
```

Comment:
you converted table like structure to list for accessible?

Reply:
why not use list from start and remove this section at all. You can style it to display horizontal?

Reply:
I removed that section